src/test/java/com/google/api/codegen/testdata/go/go_library.baseline

```diff
@@ -459,7 +476,14 @@ func (c *Client) MergeShelves(ctx context.Context, req *librarypb.MergeShelvesRe
 
 // CreateBook creates a book.
 func (c *Client) CreateBook(ctx context.Context, req *librarypb.CreateBookRequest, opts ...gax.CallOption) (*librarypb.Book, error) {
-    ctx = insertXGoog(ctx, c.xGoogHeader)
+    vals = []string{}
+    vals = append(vals, fmt.Sprintf("%s=%v", "name", req.GetName()))
+    vals = append(vals, fmt.Sprintf("%s=%v", "book.read", req.GetBook().GetRead()))
```

Comment:
I think we need the colon-equal operator to declare `vals` right?
Also consider slice literal:
```go
vals := []string{
  fmt.Sprintf("%s=%v", "name", req.GetName()),
  fmt.Sprintf("%s=%v", "book.read", req.GetBook().GetRead()),
}
```

Reply:
Done